packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -86,6 +86,35 @@ describe('createStrictAPI()', () => {
 
       expect(getByTestId('div')).toHaveCompiledCss('all', 'inherit');
     });
+
+    it('should type error when using values not the styles scope', () => {
+      const styles = css({
+        color: 'var(--ds-text)',
+        accentColor: 'red',
```

Comment:
Can you extract this test another 3 times so we can assert it errors as expected in each location?